sdk/lib/cicero/CMakeLists.txt

```diff
@@ -0,0 +1,25 @@
+
+list(APPEND SOURCE
+    cicarray.cpp
+    cicbase.cpp
+    ciccaret.cpp
+    cicfmap.cpp
+    cicimc.cpp
+    cicreg.cpp
+    cicuif.cpp)
+
+add_library(cicero_a STATIC ${SOURCE})
+add_library(cicero_w STATIC ${SOURCE})
+
+target_include_directories(cicero_a PRIVATE
```

Comment:
Should we add dependency to `psdk`?

Reply:
How does other static libraries of us manage that?

Reply:
Tested. It displays the following error message:
```txt
CMake Error at sdk/lib/cicero/CMakeLists.txt:15 (add_dependencies):
  The dependency target "psdk" of target "cicero_w" does not exist.

CMake Error at sdk/lib/cicero/CMakeLists.txt:14 (add_dependencies):
  The dependency target "psdk" of target "cicero_a" does not exist.
```

Reply:
Without `add_dependencies(cicero psdk)`, it displays the following message:

```txt
../../../sdk/lib/cicero/cicimc.cpp:10:10: fatal error: immdev.h: No such file or directory
 #include <immdev.h>
          ^~~~~~~~~~
compilation terminated.
```

Reply:
Fixed by moving `add_subdirectory(cicero)`.